pyproject.toml

```diff
@@ -25,7 +25,9 @@ dependencies = [
     "numpy<=1.26.4",
     "supervision",
     "emoji",
-    "colorama"
+    "colorama",
```

Comment:
All export deps should go in optional dependency list such as a new group say `export`